std/experimental/ndslice/internal.d

```diff
@@ -1,11 +1,333 @@
 module std.experimental.ndslice.internal;
 
+import std.range.primitives;
 import std.traits;
-import std.meta; //: AliasSeq, anySatisfy, Filter, Reverse;
+import std.meta;
+import std.experimental.ndslice.slice;
+
+/+
+fastmath do nothing here,
+but remove constraint for LDC that operations for pointer's computations
+maybe mixed up with callers computations if both computations has fastmath attribute.
+So, it is just a bridge between two fastmath functions.
+fmb alias is used to relax users.
++/
+package alias fmb = fastmath;
+
+version(LDC)
+{
+    static import ldc.attributes;
+    alias fastmath = ldc.attributes.fastmath;
+}
+else
+{
+    alias fastmath = fastmathDummy;
+}
+
+enum FastmathDummy { init }
+FastmathDummy fastmathDummy() { return FastmathDummy.init; }
```

Comment:
Perhaps nitpicks but:

1. Can't `FastmathDummy`, `fastmathDummy` and `alias fastmath` be replaced with just `enum fastmath;`
2. If `fmb` is declared as `package` should these dummy symbols be that as well, or even `private`?
3. BTW, shouldn't all public symbols be `package` in this module, since it's an internal module?